src/components/datagrid/controls/display_selector.tsx

```diff
@@ -137,8 +137,12 @@ export const useDataGridDisplaySelector = (
     },
     [lineCount]
   );
-  const setLineCountHeight = useCallback((event) => {
-    const newLineCount = Number(event.target.value);
+  const setLineCountHeight = useCallback<
+    NonNullable<EuiRangeProps['onChange']>
+  >((event) => {
+    if (!event.currentTarget) return;
+
+    const newLineCount = Number(event.currentTarget.value);
```

Comment:
Ah, I missed this on my previous review, apologies. There's no need for the extra `.value` check, `event.currentTarget.value` always exists
```suggestion
    const newLineCount = Number(event.currentTarget.value);
```

Reply:
Addressed in https://github.com/elastic/eui/pull/7012/commits/2fa936eeb565a1ef72ea85018fcd833a56f217f4